crates/wash/src/config.rs

```diff
@@ -148,7 +148,7 @@ where
     }
 
     // Environment variables with WASH_ prefix
-    figment = figment.merge(Env::prefixed("WASH_"));
+    figment = figment.merge(Env::prefixed("WASH_").split("_"));
```

Comment:
I am not quite done with the testing, but wanted to make sure this is desired. I have the feeling this is the typical way to do this, instead of using the default from `figment` (which is a dot - `.`).

Not setting this does not allow configurations via the env vars as shown in the test below (https://github.com/wasmCloud/wash/pull/97/files#diff-f7f9d11319f33b9af91da4ef6ce4b7111c1baceef1b82db506c4612ba6f808b1R356). If this is removed, the test fails as the splitting does not work on underscores.

Reply:
I will need to add a test on how this works on "multi-word" configuration options though such as here: https://github.com/wasmCloud/wash/blob/0d7cde20b51e8f253b655864c8f68cdb7cfeaee9/crates/wash/src/component_build.rs#L48